### PR TITLE
upgraded dependencies

### PR DESCRIPTION
**Pull request by BenCosfordHMRC**
https://jira.tools.tax.service.gov.uk/browse/SASS-677

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [x]  Have you checked the PR Builder passes?
- [x]  Have you checked for merge conflicts?
- [x]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: test/controllers/GetInterestsControllerSpec.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ class GetInterestsControllerSpec extends TestSuite{
   val mtditid: String = "1234567890"
   val taxYear: String = "2020"
   val service: GetInterestsService = mock[GetInterestsService]
-  val controller = new GetInterestsController(mockControllerComponents, authorisedAction, service)
+  val controller = new GetInterestsController(mockControllerComponents, authorisedAction, service)(mockExecutionContext)
```
Comment (michael-rob-poole) [inline test/controllers/GetInterestsControllerSpec.scala:35]: why did you add this in?

File: test/testUtils/TestSuite.scala [hunk 2]
```diff
@@ -47,7 +47,7 @@ trait TestSuite extends AnyWordSpec with Matchers with MockFactory with GuiceOne
   }
 
   implicit val actorSystem: ActorSystem = ActorSystem()
-  implicit val materializer: Materializer = ActorMaterializer()
+  implicit val materializer: SystemMaterializer = SystemMaterializer(actorSystem)
```
Comment (michael-rob-poole) [inline test/testUtils/TestSuite.scala:50]: Doe shis need to be here? I though we were just deleting it.

Comment (BenCosfordHMRC) [reply test/testUtils/TestSuite.scala:50]: I'm not sure really, I saw Abdullah do this and thought it was slightly better than removing it entirely but I don't think it particularly matters either way.

File: test/controllers/GetInterestsControllerSpec.scala (hunk 1 above)
Comment (BenCosfordHMRC) [reply test/controllers/GetInterestsControllerSpec.scala:35]: Thought I had an error that was saying it needed the implicit declared but it was something else in the end.
